module-gui/gui/core/BoundingBox.hpp

```diff
@@ -44,6 +44,7 @@ namespace gui
         Position pos(gui::Axis axis) const;
         std::string str() const;
         /// logical sum of bounding box by another bounding box values
+        // NOTE: This function only increases the size, position is intact so this is not really a sum
         void sum(const BoundingBox &box);
```

Comment:
maybe it's worth renaming the function if there's a need for comment?

Reply:
I renamed it to `expandSize`.